chromium_presubmit_config.json5

```diff
@@ -51,6 +51,7 @@
   // Files to skip in all checks.
   "default_files_to_skip": [
     "\\.storybook/",
+    "components/brave_rewards/browser/test/rewards_state_browsertest\\.cc",
```

Comment:
Does this skip presubmit checks for this file? Is that necessary? It would be good not to skip presubmit checking unless there's a very strong reason.

Reply:
I think we should have a relaxed form of presubmit checking for tests, as they have totally different needs than production code. Anyways, agreed.